code/modules/mob/observer/freelook/ai/eye.dm

```diff
@@ -106,3 +106,16 @@
 
 	eyeobj.acceleration = !eyeobj.acceleration
 	to_chat(usr, "Camera acceleration has been toggled [eyeobj.acceleration ? "on" : "off"].")
+
+/mob/living/silicon/ai/proc/eyepuppettoggle() //Messing with the SEE_INVISIBLE_TAG renders the lighting differently so I'd rather just swap out the icon for nothing and still render lights correctly.
```

Comment:
`/mob/living/silicon/ai/proc/ai_eye_puppet_toggle()` or `/mob/living/silicon/ai/proc/eye_puppet_toggle()` would be preferable as it keeps the naming scheme